### DISCUSSION TITLE
Livewire 3 Alpine doesn't work with Laravel Mix

### DISCUSSION THREAD

**Discussion by adriansio** [alpinejs/alpine#3768] [1. Help]
I am trying to upgrade to Livewire 3 and I can't get Alpine to work with the bundling method (I am using Laravel Mix, I can't use Vite).

This is my `webpack.mix.js`:

```
const mix = require('laravel-mix');

mix.js('resources/js/app.js', 'public/js');
```

This is my `app.js`:

```
import {Alpine, Livewire} from '../../vendor/livewire/livewire/dist/livewire.esm';
import Clipboard from '@ryangjchandler/alpine-clipboard'

Alpine.plugin(Clipboard)

Livewire.start()
```

And this is my `welcome.blade.php`:

```
<html>

<head>
    @livewireStyles

    <script src="{{ mix('js/app.js') }}" defer></script>
</head>

<body>
    <h1 x-data="{ message: 'I ❤️ Alpine' }" x-text="message"></h1>

    @livewireScriptConfig
</body>

</html>
```

After compiling with `npm run dev`, I keep getting this error:

```
Alpine Expression Error: func(...).catch is not a function
Uncaught TypeError: func(...).catch is not a function
```

Is there really no way to use Livewire 3, Alpine, and Laravel Mix with the bundling method?

**Comment by ekwoka** [comment]
As mentioned in that other discussion you found, this is an issue with your mix configuration targeting a ES versions that are fundamentally unsupported by Alpine. Alpine requires ES6 to work for many functions (proxies, async functions, etc). And having a build system that tries to convert this to ES5 will never be able to as these are not transpilable features, and if it believes it successfully managed to, it will fail to run in many cases, especially in an actual es5 environment.

**Reply by adriansio** [reply]
Thanks but is there an actual solution for this? I can't switch to Vite unfortunately.

**Reply by ekwoka** [reply]
The fix is to not be transpiling to es5

So adjust your bundler to stop targeting es5.

😊

**Reply by adriansio** [reply]
I don't think this is possible with laravel-mix unfortunately

**Reply by ekwoka** [reply]
I can't imagine that could be true.

Surely there is a setting telling it what the target is.

That would be a question for laravel-mix though.

How to change the transpilation target.

**Reply by SimoTod** [reply]
Perhaps
https://laravel-mix.com/docs/4.1/quick-webpack-configuration#quick-webpack-configuration
+
https://webpack.js.org/configuration/target/#string-1

I think Mix can also use a babelrc file if you define one. Just google it and try a few options and see which one works better for you.

**Reply by ekwoka** [reply]
could even patch laravel mix with a new target if the config doesn't allow any changes
